react-accordion-component/my-app/src/RenderTopics.js

```diff
@@ -0,0 +1,27 @@
+import { useState } from 'react';
+
+export default function RenderTopics(props) {
+  const [selected, setSelected] = useState(null);
```

Comment:
I prefer to never use `null` unless I have to, but that's just a personal preference, based on my opinion that `null` was a mistake in the language. I would use `undefined` instead.